ap/greek/templates/greek_helper/_flashcards.html

```
@@ -0,0 +1,144 @@
+<div id="flashcards" class="greek tab-pane" role="tabpanel">
+  <div class="chapters-container">
+    <div class="chapter-container">
+      <select id="fc-chapter-start" class="form-control">
+      {% for ch in chapters %}
+        <option value="{{ch}}">
+          Chapter {{ch}}
+        </option>
+      {% endfor %}
+      </select>
+    </div>
+    <div class="text-center chapter-text">
+      to
+    </div>
+    <div class="chapter-container">
+      <select id="fc-chapter-end" class="form-control" >
+      {% for ch in chapters %}
+        <option value="{{ch}}">
+          Chapter {{ch}}
+        </option>
+      {% endfor %}
+      </select>
+    </div>
+  </div>
+  <div class="flashcard-container">
+    <div id="flashcard-card">
+      <div id="flashcard-text"></div>
+    </div>
+    <div class="foot">
+      <div id="archive-placeholder">
+        <div id="archive-container">
+          <input type="checkbox" id="archive-flag"> Archive
+        </div>
+      </div>
+      <div id="flashcard-index">
+      </div>
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
+  let isFront = true;
+  let greekVocabList_fc = [];
+  let startingChapter_fc = 1;
+  let endingChapter_fc = 1;
+  let currentVocabIdx_fc = 0;
+
+  function updateIndexText() {
+    $("#flashcard-index").text(`${currentVocabIdx_fc+1}/${greekVocabList_fc.length}`);
+  }
+
+  function updateArchiveCheckbox() {
```

Not a big deal, but for that minimal performance boost you could have the function default to show, then do a check to see if you should hide.

this could apply to some of the other functions as well. I know it reads better doing it this way, but I'll leave it up to your discretion